From 9ea9d3f27686b5bccd75625c127518070d7a93e9 Mon Sep 17 00:00:00 2001
From: Alin Jerpelea <alin.jerpelea@sony.com>
Date: Thu, 9 Aug 2018 09:37:03 +0200
Subject: [PATCH] post_proc: Enable post processing for sdm660

the sdm660 SOC can use the same prost_proc code as all the other
QCOM SOCs

Test: manual - builds, boots, sound works

Change-Id: I18b9cec56c6197b4465e8009c7e50aa95e111d32
Signed-off-by: Alin Jerpelea <alin.jerpelea@sony.com>
---
 post_proc/Android.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/post_proc/Android.mk b/post_proc/Android.mk
index 66db54d..9b99d2c 100644
--- a/post_proc/Android.mk
+++ b/post_proc/Android.mk
@@ -1,4 +1,4 @@
-ifneq ($(filter msm8974 msm8226 msm8084 msm8992 msm8994 msm8996 msm8909 msm8952 msm8998 sdm845,$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(filter msm8974 msm8226 msm8084 msm8992 msm8994 msm8996 msm8909 msm8952 msm8998 sdm660 sdm845,$(TARGET_BOARD_PLATFORM)),)
 
 LOCAL_PATH:= $(call my-dir)
 
@@ -43,7 +43,7 @@ endif
 
 ################################################################################
 
-ifneq ($(filter msm8992 msm8994 msm8996 msm8909 msm8952 msm8998 sdm845,$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(filter msm8992 msm8994 msm8996 msm8909 msm8952 msm8998 sdm660 sdm845,$(TARGET_BOARD_PLATFORM)),)
 
 include $(CLEAR_VARS)
 
-- 
2.13.0

